Yoast/javascript
# Add an Alert component

## Summary


This PR can be summarized in the following changelog entry:

* [@yoast/components] Adds an Alert component.
* [@yoast/style-guide] Adds new status colors: error, info, success, warning.

## Relevant technical choices:

* Based on the [MyYoast implementation](https://github.com/Yoast/my-yoast/blob/7a3c8a6cb12cf60bfaad3dc0c1c612e43f4d59dc/client/src/components/general/Alert.js). Should be similar in usage. The cookie and dismiss should now be controlled from outside this component.
* Adds the colors to the style-guide in a hopefully reusable fashion.

## Test instructions

This PR can be tested by following these steps:

* Run the components app.
* Check the different type of alerts in the components 'tab'.
* Check the dismissable alert.
* The should look like [the design](https://github.com/Yoast/design/issues/371).
* Check in Right To Left: spacing of the icons on the correct sides.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## Screenshot
<img width="983" alt="alerts" src="https://user-images.githubusercontent.com/35524806/64234153-fbe66300-cef5-11e9-8262-1e44e1a94467.png">

## UI changes

## Quality assurance


Partially fixes: https://github.com/Yoast/wordpress-seo/issues/13173
